src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao)
+                injector.getInstance(ManufacturerDao.class);
+        Manufacturer firstManufacturer = new Manufacturer();
+        firstManufacturer.setName("Lexus");
+        firstManufacturer.setCountry("USA");
+
+        manufacturerDao.create(firstManufacturer);
+
+        Manufacturer secondManufacturer = new Manufacturer();
+        secondManufacturer.setName("Audi");
+        secondManufacturer.setCountry("Germany");
+
+        manufacturerDao.create(secondManufacturer);
 
+        System.out.println(manufacturerDao.get(11L));
```

Comment:
```suggestion
        System.out.println(manufacturerDao.get(secondManufacturer.getId()));
```

It's better to make code more universal